### ISSUE TITLE
Shortcut button focus is not visible in high contrast mode

### ISSUE THREAD

**Issue by micahgodbolt** [storybookjs/storybook#13373]
1. go to https://next--storybookjs.netlify.app/official-storybook/?path=/story/addons-a11y-basebutton--default
2. turn on high contrast mode in windows
3. turn of "focus" element state
4. note that there is no visible focus indicator

![image](https://user-images.githubusercontent.com/1434956/101105101-94e4ad80-3581-11eb-8f91-70f7d1ebe965.png)

**Comment by bmeersma** [comment]
Hi @micahgodbolt do you have an update on this issue?

**Comment by shilman** [comment]
ZOMG!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.2.0-beta.4 containing PR #13699 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb upgrade --prerelease
```

Closing this issue. Please re-open if you think there's still more to do.
